### ISSUE TITLE
Don't practically export the map

### ISSUE THREAD

**Issue by inca11** [Pottus/Texture-Studio#83]
So..i can open my map by using /loadmap , but when i want to export it,the game shows me that it's actually working,but it does not.
Photos: https://imgur.com/a/nxjAQvu
Hope that i will get an answer so i can export my map because i love it.
Map photos: https://imgur.com/a/YSb2xnv

**Comment by Crayder** [comment]
Is the file created at all?

**Comment by Crayder** [comment]
So the file isn't created? The file manager plugin should say something if it can't open a file.

**Comment by inca11** [comment]
No..it does not says anything..i am exporting the map but the file does not create in Exportmap,even tho it shows me the dialog "The map has been exported..." and when i look in the console i see that error..

**Comment by inca11** [comment]
Soo...can u help me or something? i really want that map
